docs/source/notebooks/mmm/mmm_build_from_yml_example.ipynb

```diff
@@ -0,0 +1,1891 @@
+{
```

Comment:
<p>Let's use the <code>from pymc_marketing.paths import data_dir</code> here</p><p> </p><p><code>from pymc_marketing.paths import data_dir</code></p><p><code>X = pd.read_csv(data_dir / "processed" / "X.csv")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>